mqtt.md

```diff
@@ -253,27 +262,33 @@ jq .
   - `word` => you need to copy this verbatim
 
 <!--v-->
-## Command
+
```

Comment:
As you've been meaning to fix all white lines in the file. I think this one is incorrect?